### DISCUSSION TITLE
How to Reset Progress?

### DISCUSSION THREAD

**Discussion by kmanadkat** [greatfrontend/greatfrontend#19] [Ideas/Feature Requests]
I would like to start fresh from beginning and I was wondering if it is possible to reset my progress on the site. Thanks.

**Comment by gfeeng** [comment]
There's currently no easy way to do so and the only way is to go to each question and reset. We'll prioritize this feature.

**Reply by kmanadkat** [reply]
Thanks for writing back. I was not aware that I can mark a question as not complete, individually. Just discovered that feature
![Screenshot 2023-01-02 at 6 19 43 PM](https://user-images.githubusercontent.com/25281055/210233682-91b7af0c-fb3d-4aec-a1da-e95f1e56c188.png)


Anyways, it is good to reset entire progress in one go. All major coding / learning platforms have it. Thanks for understanding.

**Comment by yangshun** [comment]
This has been done! You can reset progress from the profile page

<img width="890" alt="image" src="https://user-images.githubusercontent.com/1315101/236369135-84894af8-ae2f-46bb-8ccb-8adcb9c869ee.png">
